package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-influential",
-  "version": "3.1.0",
+  "version": "4.0.0",
```

Comment:
This project uses `npm` to manage releases, so please don't update the version manually. After this merges I'll run `npm verison major`, which will bump the version, commit, tag, and I don't remember if there's anything else and I'll publish the update.